public/js/studentProject.js

```diff
@@ -0,0 +1,52 @@
+
+const addStd = document.getElementById('addStd');
+const addStudent = document.getElementById('addStudent');
+const selector = document.getElementById('selector');
+const URL = window.location.href;
+const splitUrl = URL.split('/');
+const cohortId = splitUrl[splitUrl.length - 1];
+const projectId = splitUrl[splitUrl.length - 4];
+const deleteStudentButton = document.querySelectorAll('.delete');
+const projectType = splitUrl[splitUrl.length - 5];
+
+
+let projectTypeValue = '';
+if (projectType.toLowerCase() === 'clients') {
+  projectTypeValue = 0;
+} else if (projectType.toLowerCase() === 'community') {
+  projectTypeValue = 1;
+}
+
+const addStudents = () => {
+  addStd.classList.toggle('sectionAddstd--visible');
+};
+
+deleteStudentButton.forEach((button) => {
+  const deleteProjectData = { std_project: button.id };
```

Comment:
Rename `std_project` to `studentProject`.

Reply:
And anyway, how does it make sense to assign `studentProject` the value of `button.id`? What are you doing here?

Reply:
@lubnaabd I dug up what you were trying to achieve, firstly, line 9 `const deleteStudentButton = document.querySelectorAll('.delete');` isn't selecting any buttons, it is selecting anchor elements. So it is either you change your anchors to buttons, or change the variable's name to `deleteStudentAnchors` and act accordingly, then you're saying:
```js
const deleteProjectData = { std_project: button.id };
```
What is `std_project`? in your `src/views/cohortStudents.hbs` on line 28 you've said:
```hbs
<a class="delete" id="{{std_id}}">Delete</a>
```
Which by the way needs to be renamed to `studentId`.
You're passing here a student's ID, then assigning it to `std_project`? Why? What made sense when you've made this?